Backport from Fedora Project.
https://src.fedoraproject.org/rpms/libharu

Fix the install destination of some files.

Upstream-Status: inappropriate

Signed-off-by: Zheng Ruoqin <zhengrq.fnst@cn.fujitsu.com>

diff -urp libharu-RELEASE_2_3_0/CMakeLists.txt libharu-RELEASE_2_3_0_cmake/CMakeLists.txt
--- libharu-RELEASE_2_3_0/CMakeLists.txt	2013-10-30 20:11:59.000000000 +0700
+++ libharu-RELEASE_2_3_0_cmake/CMakeLists.txt	2017-06-01 15:56:06.533552795 +0700
@@ -25,7 +25,7 @@ endif(MSVC)
 
 # information about libharu
 set(LIBHPDF_MAJOR 2)
-set(LIBHPDF_MINOR 2)
+set(LIBHPDF_MINOR 3)
 set(LIBHPDF_PATCH 0)
 set(LIBHPDF_VERSION ${LIBHPDF_MAJOR}.${LIBHPDF_MINOR}.${LIBHPDF_PATCH})
 set(LIBHPDF_DESCRIPTION "libHaru is a free, cross platform, open source library for generating PDF files.")
@@ -210,9 +210,9 @@ set(
 install(FILES ${haru_HDRS} DESTINATION include)
 
 # install various files
-install(FILES README CHANGES INSTALL DESTINATION .)
+install(FILES README CHANGES INSTALL DESTINATION ${SHARE_INSTALL_PREFIX}/libharu)
 if(NOT DEVPAK)
-  install(DIRECTORY if DESTINATION .)
+  install(DIRECTORY if DESTINATION ${SHARE_INSTALL_PREFIX}/libharu)
 endif(NOT DEVPAK)
 if(DEVPAK)
   install(FILES ${CMAKE_BINARY_DIR}/libharu.DevPackage DESTINATION .)
diff -urp libharu-RELEASE_2_3_0/src/CMakeLists.txt libharu-RELEASE_2_3_0_cmake/src/CMakeLists.txt
--- libharu-RELEASE_2_3_0/src/CMakeLists.txt	2013-10-30 20:11:59.000000000 +0700
+++ libharu-RELEASE_2_3_0_cmake/src/CMakeLists.txt	2017-06-01 16:08:45.425105746 +0700
@@ -90,11 +90,13 @@ if(LIBHPDF_SHARED)
   target_link_libraries(${LIBHPDF_NAME} ${ADDITIONAL_LIBRARIES})
   if(WIN32 AND NOT CYGWIN)
   	set_target_properties(${LIBHPDF_NAME} PROPERTIES DEFINE_SYMBOL HPDF_DLL_MAKE)
+  else(WIN32 AND NOT CYGWIN)
+  	set_target_properties(${LIBHPDF_NAME} PROPERTIES SOVERSION ${LIBHPDF_VERSION})
   endif(WIN32 AND NOT CYGWIN)
   install(
     TARGETS ${LIBHPDF_NAME}
-    ARCHIVE DESTINATION lib
-    LIBRARY DESTINATION lib
+    ARCHIVE DESTINATION ${LIB_INSTALL_DIR}
+    LIBRARY DESTINATION ${LIB_INSTALL_DIR}
     RUNTIME DESTINATION bin
   )
 endif(LIBHPDF_SHARED)
